docs/DeveloperGuide.md

```diff
@@ -391,6 +391,18 @@ class PersonBuilder {
 
 All our existing tests using `PersonBuilder` will still work, and all we had to do was update definitions in `PersonBuilder`.
 
+### Show attendance
+
+Shows if a student is present on a specific day.
+
+![ShowAttendanceSequenceDiagram](images/ShowAttendanceSequenceDiagram.png)
```

Comment:
Would be good to move the diagram below the sequence of action, as per our docs structure